### ISSUE TITLE
Daily difference for "Recovered", "Deaths", and "Tested" charts incorrectly showing as "(+0)"

### ISSUE THREAD

**Issue by bowdentom** [reustle/covid19japan#429]
<img width="1202" alt="Screen Shot 2020-07-31 at 17 42 48" src="https://user-images.githubusercontent.com/17093880/89018313-a52c1800-d356-11ea-920f-a0ec5d441722.png">

The source data from Google Sheets shows the following:

<img width="723" alt="Screen Shot 2020-07-31 at 17 53 51" src="https://user-images.githubusercontent.com/17093880/89018515-f805cf80-d356-11ea-9e7a-9377fd216617.png">

The error appears to be in the JSON api file at https://data.covid19japan.com/summary/latest.json
which is showing:

<img width="365" alt="Screen Shot 2020-07-31 at 17 58 40" src="https://user-images.githubusercontent.com/17093880/89018906-81b59d00-d357-11ea-8c50-f88c04b035a3.png">

@reustle

**Comment by liquidx** [comment]
This is working as intended. We hadn't added the numbers for 2020-07-31 (the latest one was in 2020-07-30 in your screenshot), so we can't show any increases until we actually add the numbers

**Comment by bowdentom** [comment]
I don’t think this should be the intended working behavior. The user will look at the difference number, and conclude that there has been no change since the previous day. In fact, there has been a change since the previous day, so showing "(+0)" is plain incorrect.

**Comment by vlumi** [comment]
I agree with @bowdentom. Since the current day's numbers will be tentative and incomplete for the majority of the time, I feel that they should not be promoted so much. Wouldn't it make more sense to keep the confirmed, final previous day's numbers in the spotlight instead, and push the tentative numbers a bit down?

**Comment by liquidx** [comment]
Fair point, ok, how about if we show yesterday's increase then until there's data for the current day? Will that be better?

**Comment by reustle** [comment]
@liquidx Maybe this can be split into 2 issues?

1. If not done already, we can not show the difference if it is 0, so `(+0)` goes away

2. Move the other half of this issue (regarding the data) to the data repo
